dist/videojs-offset.js

```diff
@@ -57,18 +57,22 @@ var vjsoffset = function(options) {
   }
 
   this.on('timeupdate', function(){
-    var curr = this.currentTime();
-    if(curr < 0){
-      this.currentTime(0);
-      this.play();
-    }
-    if(this._offsetEnd > 0 && (curr > (this._offsetEnd-this._offsetStart))) {
-      this.pause();
-      if (!this._restartBeginning) {
-        this.currentTime(this._offsetEnd-this._offsetStart);
-      } else {
-        this.trigger('loadstart');
+    var update_time = new Date();
+    if (this.lustUpdateTime==undefined || update_time-this.lustUpdateTime>1000) {
+      var curr = this.currentTime();
+      this.lustUpdateTime = new Date();
+      if (curr < 0) {
         this.currentTime(0);
+        this.play();
+      }
+      if (this._offsetEnd > 0 && (curr > (this._offsetEnd - this._offsetStart))) {
+        this.pause();
```

Comment:
Leave only this line with `this.pause();` and remove the rest of this if-branch.
As discussed, we don't need it.